docs/sophia_stdlib.md

```diff
@@ -39,6 +39,7 @@ include "List.aes"
 - [Triple](#Triple)
 - [BLS12_381](#BLS12_381)
 - [Frac](#Frac)
+- [Set](#Set)
```

Comment:
This actually points to Bits `set`

Reply:
Uh, it was always a mystery for me how do the labels in md work

Reply:
This linking issue should be fixed in this commit 028b12ddc6c6a478207744057b6817b4abdfe0b2